website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-model-timing-tab.md

```diff
@@ -4,6 +4,9 @@ id: "cloud-model-timing-tab"
 ---
 
 ### Overview
+
+Accounts on the _Team_ and _Enterprise_ plans see the model timing.
```

Comment:
Could you make this into a callout section above the Overview like on the [environment variables page](https://docs.getdbt.com/docs/dbt-cloud/using-dbt-cloud/cloud-environment-variables))?

It should say that ``` Model Timing is only available on the Team and Multi-tenant Enterprise plans```.

Reply:
done